{"schemaVersion":{"patch":0,"major":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError"]]},"metadata":{"role":"symbol","externalID":"s:5FIDO224PublicKeyCredentialErrorO16errorDescriptionSSSgvp","title":"errorDescription","symbolKind":"property","modules":[{"name":"FIDO2"}],"roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"? { "},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}],"languages":["swift"]}]}],"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/fido2\/publickeycredentialerror\/errordescription"]}],"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"text":".","type":"text"}],"sections":[],"identifier":{"url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/errorDescription","interfaceLanguage":"swift"},"references":{"doc://com.ibm.verify.ios.sdk/documentation/FIDO2":{"abstract":[],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2","url":"\/documentation\/fido2","title":"FIDO2","kind":"symbol","role":"collection","type":"topic"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError/errorDescription":{"url":"\/documentation\/fido2\/publickeycredentialerror\/errordescription","kind":"symbol","title":"errorDescription","type":"topic","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"abstract":[],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError\/errorDescription"},"doc://com.ibm.verify.ios.sdk/documentation/FIDO2/PublicKeyCredentialError":{"type":"topic","role":"symbol","url":"\/documentation\/fido2\/publickeycredentialerror","abstract":[{"text":"A type that indicates when Fido2ApiClient encounters an error.","type":"text"}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/FIDO2\/PublicKeyCredentialError","title":"PublicKeyCredentialError","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"PublicKeyCredentialError","kind":"identifier"}],"navigatorTitle":[{"text":"PublicKeyCredentialError","kind":"identifier"}]}}}